atom/browser/net/resolve_proxy_helper.cc

```diff
@@ -75,8 +75,8 @@ void ResolveProxyHelper::OnProxyLookupComplete(
   if (proxy_info)
     proxy = proxy_info->ToPacString();
 
-  if (!completed_request.callback.is_null())
-    completed_request.callback.Run(proxy);
+  if (!std::move(completed_request.callback).is_null())
+    std::move(completed_request.callback).Run(proxy);
```

Comment:
hm, this doesn't make sense, you can't move it twice.

Reply:
oh derp yeah: this may need to be a RepeatingCallback then?